project-base/docker/php-fpm/Dockerfile

```diff
@@ -77,3 +76,48 @@ RUN if [ -n "$github_oauth_token" ]; then composer config -g github-oauth.github
 
 # hirak/prestissimo makes the install of Composer dependencies faster by parallel downloading
 RUN composer global require hirak/prestissimo
+
+########################################################################################################################
+
+FROM base as production
+
+ARG project_root=.
+
+COPY --chown=www-data:www-data / /var/www/html
+COPY --chown=www-data:www-data ${project_root}/app/config/parameters.yml.dist /var/www/html/${project_root}/app/config/parameters.yml
+COPY --chown=www-data:www-data ${project_root}/app/config/domains_urls.yml.dist /var/www/html/${project_root}/app/config/domains_urls.yml
```

Comment:
update installation document for production with docker

Reply:
hmmmmm `docker-compose logs  webserver ` produce just `shopsys-framework-webserver| Error grabbing logs: EOF` and doesn't log anything in development mode. But this could be different issue that is not related to this PR changes